tensorflow/compiler/aot/codegen.cc

```diff
@@ -229,8 +228,9 @@ Status GenResultMethods(const tf2xla::Config& config,
   int readonly_variables = absl::c_count_if(
       config.variable(),
       [](const tf2xla::Variable& var) { return var.readonly(); });
-  if (config.fetch_size() + config.variable_size() - readonly_variables !=
-      num_results) {
+  const size_t actual_num_results = config.fetch_size()
+                                 + config.variable_size() - readonly_variables;
```

Comment:
We don't need to change this to `size_t` now, right?

Reply:
Line 231: `size_t` change to `int`